src/seedu/addressbook/common/Messages.java

```diff
@@ -15,4 +15,5 @@
             "java seedu.addressbook.Main [STORAGE_FILE_PATH]";
     public static final String MESSAGE_WELCOME = "Welcome to your Address Book!";
     public static final String MESSAGE_USING_STORAGE_FILE = "Using storage file : %1$s";
+    public static final String MESSAGE_FEATURE_IN_PROGRESS = "This feature is in progress";
```

Comment:
Good job implementing the two features! Just wanted to clarify though, when will this message be displayed?